cmd/validators.go

```diff
@@ -130,6 +133,9 @@ func inferArgumentLocation(arg string) common.Location {
 				return common.ELocation.BlobFS()
 			case strings.Contains(host, benchmarkSourceHost):
 				return common.ELocation.Benchmark()
+				// enable targeting an emulator/stack
+			case IPv4Regex.MatchString(host):
+				return common.ELocation.Blob()
```

Comment:
There's no guarantee that it's Blob. 

Btw we should cover custom domains too: a specified URL (foobar.com) could point to blob/file/ADLS endpoints. I believe the right solution in this case is to take what the user has given as `--from-to` and discard the inference.